### ISSUE TITLE
Feature: Possibility to use terminationGracePeriodSeconds for Pods / Jobs for kubernetes

### ISSUE THREAD

**Issue by klopfdreh** [spring-cloud/spring-cloud-deployer#495]
We faced some issues that the Composed Task Runner of Spring Cloud Data Flow Server received a SIGTERM and was shutdown gracefully inbetween DSL operations. The CTR then returned a non-error return code (0).

It would be great to set the `terminationGracePeriodSeconds` to a very low value (e.g. 1. - default is 30) so that the container is killed directly. In those cases we can detect the crash and use our restart mechanism.

See: 
* https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#pod-termination
* https://kubernetes.io/docs/concepts/workloads/controllers/job/#termination-of-job-pods

this could be configured for SCDF here like described here:

https://docs.spring.io/spring-cloud-dataflow/docs/current/reference/htmlsingle/#configuration-kubernetes-deployer

**Comment by corneil** [comment]
Are you launching with `createJob` true or false?

**Comment by klopfdreh** [comment]
Yes we are using createJob true - but from what I saw in the docs of Kubernetes it applies to Pods and Pods created by Jobs.
